example.json

```diff
@@ -48,5 +48,10 @@
 
     // to be added to the _extensions object in course.json
     "_languagePicker": {
-        "navigationBarLabel" : "Select course language"
+        "navigationBarLabel" : "Select course language",
+        "languageSelector": "Language selector",
+        "_navTooltip": {
+            "_isEnabled": true,
+            "text": "Language Picker"
```

Comment:
Is it a language picker or a language selector?

Reply:
@oliverfoster languageSelector is in the templates and was in schemas but not example, happy to address in this PR though.

Reply:
I don't think you should change the schema property names. I meant that the text is both `"Language Picker"` and `"Language selector"`. You've added `"Language Picker"` but the original text says `"Language selector"`.

Reply:
@oliverfoster do we want to leave it as Language Selector and update tooltip to match?

Reply:
Yea, probably easier.